ecs-cli/vendor/github.com/aws/aws-sdk-go/service/ecs/api.go

```diff
@@ -6559,6 +6560,15 @@ func (s RegisterTaskDefinitionInput) GoString() string {
 	return s.String()
 }
 
+// GoString returns deterministic string representation
+func (s RegisterTaskDefinitionInput) SortedGoString() (string, error) {
```

Comment:
The vendor directory is auto-generated using dependencies specified in godep, so any thing can be overwritten during package upgrade.

I would recommend putting this in `/ecs-cli/modules/compose/ecs/utils/convert_task_definition.go`